src/System.Runtime.WindowsRuntime/src/System/IO/StreamOperationAsyncResult.cs

```diff
@@ -150,7 +150,12 @@ internal bool CancelStreamOperation()
             if (_callbackInvoked)
                 return false;
 
-            _asyncStreamOperation.Cancel();
+            try
+            {
+                _asyncStreamOperation?.Cancel();
+            }
+            catch { }
```

Comment:
> NullReferenceException on that field.

Now that you've added the null check, there won't be a null ref from accessing that field.

> Would you like me to remove the try-catch block?

Yes, please.  Until there's good reason to do so, let's let any unexpected exceptions propagate.